src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/repository/DisputeRepository.java

```diff
@@ -21,5 +22,8 @@ public interface DisputeRepository extends CrudRepository<Dispute, UUID> {
 
 	/** Fetch all records whose assignedTs has a timestamp older than the given date. */
     public Iterable<Dispute> findByAssignedTsBefore(Date olderThan);
+    
+    /** Fetch all records which have the specified jjAssigned. */
+    public List<Dispute> findByJjAssignedIgnoreCase(String jjAssigned);
```

Comment:
Will this get moved to the JJDisputeRepository?

Reply:
Yes, it will when we have the JJDisputeRepository created.